packages/design-system/src/components/HelpDrawer/useHelpDrawerAnalytics.ts

```diff
@@ -39,7 +39,6 @@ export default function useHelpDrawerAnalytics({
   }
 
   const [headingRef] = useAnalyticsContent({
-    componentName: 'Dialog',
```

Comment:
Your changes here made me realize that in [this change](https://github.com/CMSgov/design-system/pull/1944) not only did I fail to remove the unused `componentName` property, but I also mislabeled this component through copypasta. Since this is in a fork, would you mind [updating this string for me](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/components/HelpDrawer/useHelpDrawerAnalytics.ts#L27) to reflect that it's for the help drawer?